monit/files/modules

```diff
@@ -1,50 +1,17 @@
-# -*- coding: utf-8 -*-
-# vim: ft=jinja
-
-{#- This sequence of loops is a bit harder to mantain. So in order
-    to make it a bit easier, the comments are following the example 
-    below line by line.
-
-    modules:
-      nginx_init:
-        process: 
-          custom:
-            name: nginx
-          with:
-            pidfile: /var/run/nginx.pid
-          config:
-            group: www
-            start: "/etc/init.d/nginx start"
-            stop: "/etc/init.d/nginx stop"
-          if: 
-            failed: 127.0.0.1 port 80 protocol http
-            action: restart
--#}
```

Comment:
Likewise, is there a reason for removing all of these comments from this file?

Reply:
Same story. I don't want to mess with that file as I don't understand it properly.